### PR TITLE
CLI: add congestion command

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: sdk/src/types/api/core.rs [hunk 1]
```diff
@@ -325,7 +325,7 @@ pub struct CongestionResponse {
     pub reference_mana_cost: u64,
     /// The Block Issuance Credits of the requested account.
     #[serde(with = "string")]
-    pub block_issuance_credits: u64,
+    pub block_issuance_credits: i128,
```
Comment (thibault-martinez) [inline sdk/src/types/api/core.rs:328]: Not too sure about this but it can actually be negative...

**Review by Thoralf-M — DISMISSED** [review]

File: cli/src/wallet_cli/mod.rs [hunk 2]
```diff
@@ -478,6 +480,28 @@ pub async fn claimable_outputs_command(wallet: &Wallet) -> Result<(), Error> {
...
+        .or(wallet_data
+            .accounts()
+            .next()
+            .map(|o| o.output.as_account().account_id_non_null(&o.output_id)))
+        .or(wallet_data
+            .implicit_accounts()
+            .next()
+            .map(|o| AccountId::from(&o.output_id)))
+        .ok_or(WalletError::NoAccountToIssueBlock)?;
+    drop(wallet_data);
```
Comment (DaughterOfMars) [inline cli/src/wallet_cli/mod.rs]: Just a tiny bit better I think
```suggestion
    let account_id = {
      let wallet_data = wallet.data().await;
      account_id
          .or_else(|| wallet_data
              .accounts()
              .next()
              .map(|o| o.output.as_account().account_id_non_null(&o.output_id)))
          .or_else(|| wallet_data
              .implicit_accounts()
              .next()
              .map(|o| AccountId::from(&o.output_id)))
          .ok_or(WalletError::NoAccountToIssueBlock)?
    };
```

File: sdk/src/types/api/core.rs (hunk 1 above)
Comment (DaughterOfMars) [reply sdk/src/types/api/core.rs:328]: 👀

**Review by Thoralf-M — APPROVED** [review]

File: sdk/src/types/api/core.rs (hunk 1 above)
Comment (thibault-martinez) [reply sdk/src/types/api/core.rs:328]: We still need the full amplitude of an u64 but it can also go negative so I guess i128 will do

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**